### PR TITLE
Enabling the extraction of fMRI motion correction parameters

### PR DESCRIPTION
**Pull request by benjamindeleener**
This Pull Request enables the extraction of the motion correction parameters when using `sct_fmri_moco`. Extracting the motion correction parameters are useful when using them as regressors in fMRI analyses.

The proposed additional outputs are:
- a time-series with 1 voxel in the XY plane, for the X and Y motion direction (two separate files), as required for FSL analysis.
- a TSV file with the slice-wise average of the motion correction for XY (one file), that can be used for Quality Control.

The floor is open to discussion.

### CODE REVIEW

**Comment by jcohenadad** [comment]
added a discussion thread in the [forum](http://forum.spinalcordmri.org/t/enabling-the-extraction-of-fmri-motion-correction-parameters/94) to gather more input

File: scripts/sct_fmri_moco.py [hunk 1]
```diff
@@ -62,6 +62,7 @@ def __init__(self):
         self.iterAvg = 1  # iteratively average target image for more robust moco
         self.num_target = '0'
         self.is_sagittal = False  # if True, then split along Z (right-left) and register each 2D slice (vs. 3D volume)
+        self.output_motion_param = True  # if True, the motion parameters are outputed
```
Comment (jcohenadad) [inline scripts/sct_fmri_moco.py]: typo: outputted

**Comment by jcohenadad** [comment]
@benjamindeleener are you still working on this branch?

**Comment by benjamindeleener** [comment]
@jcohenadad I was waiting for feedback. I'll update the documentation.

**Comment by jcohenadad** [comment]
@benjamindeleener release in ~1d. Now is your chance ;-)

**Pull request merged**